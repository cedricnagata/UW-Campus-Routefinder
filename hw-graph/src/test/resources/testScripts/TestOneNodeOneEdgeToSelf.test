# Create a graph
CreateGraph Graph

# Add a node
AddNode Graph n1

# Add an edge from node to same node
AddEdge Graph n1 n1 e1

# Print nodes in the graph and children of n1
ListNodes Graph
ListChildren Graph n1