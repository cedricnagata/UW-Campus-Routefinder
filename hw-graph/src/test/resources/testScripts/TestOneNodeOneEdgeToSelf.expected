# Create a graph
created graph Graph

# Add a node
added node n1 to Graph

# Add an edge from node to same node
added edge e1 from n1 to n1 in Graph

# Print nodes in the graph and children of n1
Graph contains: n1
the children of n1 in Graph are: n1(e1)